Sources/ArgumentParser/Utilities/StringExtensions.swift

```diff
@@ -144,4 +144,38 @@ extension String {
       self[range].lowercased()
     }.joined(separator: String(separator))
   }
+  
+  func editDistance(to target: String) -> Int {
```

Comment:
Maybe document that this is using [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)(as per your description in this PR)? :) 

Also can we maybe come up with a better naming? I think `fuzzyDistance:to`, `measuredSimilarity:with`, `measuredDifference:with` are a bit more clear, at least to me.

```swift
"bar".measuredSimilarity(with: "baz")
```
reads out better than:
```swift
"bar".fuzzyDistance(to: "baz")
```

Reply:
A comment about the algorithm would be great, but I think the name is fine here — "edit distance" is a common term for this algorithm.

Reply:
@natecook1000 my thoughts exactly. I'll add the comment :)

Reply:
Comment added.